frontend/src/components/BaseDashboard/pages/Reports/index.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { Table, Thead, Tbody, Tr, Th, Td } from "react-super-responsive-table";
+import "react-super-responsive-table/dist/SuperResponsiveTableStyle.css";
```

Comment:
why not react-table? I don't mind this lib (although it's not very popular), but I'm curious

Reply:
because it's responsive, try to resize window to mobile width @tudorgergely